src/test/java/org/opensearch/security/ssl/SecuritySSLReloadCertsActionTests.java

```diff
@@ -186,6 +201,99 @@ public void testReloadHttpSSLSameCertsPass() throws Exception {
         assertReloadCertificateSuccess(rh, "http", getInitCertDetailsExpectedResponse());
     }
 
+    @Test
+    public void testReloadHttpCertDifferentTrustChain_skipDnValidationPass() throws Exception {
```

Comment:
nit: can we add a test to verify default behavior, this is to ensure default doesn't get changed in future without explicit test modification.

Reply:
Absolutely, good suggestion

Reply:
Added @shikharj05!